### ISSUE TITLE
Channel list is too wide and cannot be resized

### ISSUE THREAD

**Issue by tkzv** [hexchat/hexchat#1386]
With font size 18 (I'm using Liberation family) and screen DPI 96 the list of channels is about 1600 pixels wide and cannot be made any narrower. Furthermore, if it is a part of the main window, the main window also cannot be narrowed. Looks like it's the "Topic" column that limits the minimal width.

**Comment by TingPing** [comment]
The chan list has no min size here.. I can make it 0px wide.

> Furthermore, if it is a part of the main window,

How can it not be part of the main window?

**Comment by tkzv** [comment]
How do you make it 0 px wide? When I drag its borders, they stop at some point and move no further. Same in KDE and LXDE.

However, the limit turns out to depend on the locale. With my font settings and for en_US it's 1085 for detached Channel List and 1099 for the main HexChat window with ChanList docked. The above ~1600 value was for ru_RU locale. This is the width when HexChat starts.  As I said above, it seems to be determined by the minimum width of Topic column — it cannot be narrowed below a certain limit, ~870 pixels for en_US, ~1240 for ru_RU.

To detach Channel List right-click ChanList tab and select Detach from the context menu. There is an option for it to be detached by default: Settings -> Preferences, Interface -> Channel Switcher, set "Open utilities in" = Windows.

**Comment by TingPing** [comment]
Ah misunderstanding, I thought you mean the channel tree in the main interface not the channel list dialog.

That dialog does have a minimum size for me but it seems to be a reasonable ~480/260px.

**Comment by tkzv** [comment]
The limit depends on symbol size and X resolution.

**Comment by TingPing** [comment]
Yes so whats the problem exactly? Its not like you can use it at smaller than that?

**Comment by tkzv** [comment]
1600 window on a 1440 screen. And smaller font stains my eyes.
